pyproject.toml

```diff
@@ -0,0 +1,84 @@
+[tool.black]
+line-length  =  100
+target-version  =  [
+    'py37',
+]
+include  =  '(bluepysnap|tests)\/.*\.py'
+
+[tool.isort]
+profile  =  'black'
+line_length  =  100
+known_local_folder  =  [
+'utils',
+'test_module',
+]
+
+[tool.pylint.messages_control]
+disable = [
+    'bad-continuation',
+    'fixme',
+    'invalid-name',
+    'len-as-condition',
+    'no-else-return',
+    'ungrouped-imports',
+    'useless-object-inheritance',
```

Comment:
`ungrouped-imports` and `useless-object-inheritance` seem unnecessary.

Reply:
I'll clear them and see what is the output.